ddtrace/ddtrace.go

```diff
@@ -171,6 +171,15 @@ type StartSpanConfig struct {
 	Context context.Context
 }
 
+func NewStartSpanConfig(opts ...StartSpanOption) StartSpanConfig {
+	var cfg StartSpanConfig
+	for _, fn := range opts {
+		fn(&cfg)
+	}
+
```

Comment:
```suggestion
```
Suggestion based on the style guidelines https://github.com/DataDog/dd-trace-go/wiki/Style-guidelines#general-guidelines